index.html

```diff
@@ -91,42 +107,52 @@ <h2 class="project-title">Tonic</h2>
               </div>
             </div>
           </div>
-          <article class="card">
-            <figure>
-              <img src="images/big_icon/multi.png" alt=" " />
-            </figure>
-            <div class="left-block">
-              <div class="project-header">
-                <h2 class="project-title">  Multi-Post Stories</h2>
-                <div class="header-details">
-                  <p class="client">CANOPY</p>
-                  <span class="seprator-circle"></span>
-                  <p class="role">Back End Dev</p>
-                  <span class="seprator-circle"></span>
-                  <p class="year">2015</p>
+          <div class="multi-post-stories">
```

Comment:
- As suggested by the previous reviewer, I kindly suggest that you fix your work section as it's not aligning properly
- [ ] Your styles for mobile are not showing. please also ensure your site is still mobile responsive after applying desktop styles.
- [ ] Kindly ensure that your styles for desktop stay on desktop and do not override the mobile styling



<img width="467" alt="image" src="https://user-images.githubusercontent.com/52934128/201454414-b8cc11b8-201b-478d-9c81-9dadf2ecdd00.png">

- [ ] Your contact section items on mobile should be on top of each other vertically as seen in the mobile design from Figma.

| Figma Design  | Your Design |
| ------------- | ------------- |
|<img width="148" alt="image" src="https://user-images.githubusercontent.com/52934128/201454588-806bfa90-5264-452f-94df-cb7ca47cdb4b.png">  | <img width="305" alt="image" src="https://user-images.githubusercontent.com/52934128/201454602-e15bc9ae-3bff-482d-8dc6-62dab1bdc71c.png"> |